plugins/sentinelone/komand_sentinelone/util/helper.py

```diff
@@ -83,10 +83,21 @@ def _wrapper(self, *args, **kwargs):
 
 def format_subdomain(instance: str) -> str:
     """
-    If an input subdomain contains a scheme or the Sentinelone secondlevel domain, strip these values
+    If an input subdomain contains a scheme or the SentinelOne second-level domain, strip these values
     """
-    instance = instance.replace(".sentinelone.net", "")
-    return instance.replace("https://", "").replace("http://", "")
+
+    # Remove the scheme if it exists
+    if instance.startswith("http://"):
+        instance = instance[len("http://") :]
+    elif instance.startswith("https://"):
+        instance = instance[len("https://") :]
```

Comment:
Looks good 😄 If you want to refactor it or something (it's up to you) I think this could be looped 😄

Reply:
I'll just keep it how it is since I can't see us needing to extend the amount of schemes we loop through in the future 😄